app/initial_records.py

```diff
@@ -0,0 +1,72 @@
+default_users = [
```

Comment:
create a more granular folder structure for the project. If you are using an MVC pattern then split parts for model, view, and controller in separate folders and mode all the 'model' related things in that folder